src/core/proj/qgscoordinatereferencesystem.cpp

```diff
@@ -3079,6 +3079,25 @@ QgsCoordinateReferenceSystem QgsCoordinateReferenceSystem::toGeographicCrs() con
     if ( !geoCrs )
       return QgsCoordinateReferenceSystem();
 
+    const PJ_TYPE pjType = proj_get_type( geoCrs.get( ) );
+    if ( pjType == PJ_TYPE_GEOCENTRIC_CRS )
+    {
+      // special case: proj_crs_get_geodetic_crs considers a geocentric crs as already a geographic CRS, whereas
```

Comment:
this comment is not accurate. A geodetic CRS is either a geocentric CRS, a geographic CRS or other uncommon geodetic CRS (spherical planetocentric)

Reply:
Thanks!

Would "special case: while a geocentric crs IS a geodetic CRS, this particular QGIS method advertises that it will always return a geographic latitude/longitude based CRS. So we build a geographic CRS using the same datum as the original CRS" 

be more accurate?

Reply:
> be more accurate?

yes sounds perfect

Reply:
Thanks, applied